.pipeline/checkstyle-suppressions.xml

```diff
@@ -10,6 +10,5 @@
   <suppress files="[/\\]core[/\\]model[/\\]" checks=".*"/>
   <suppress files="[/\\]orchestration[/\\]model[/\\]" checks=".*"/>
   <!-- Suppress TODOs -->
-  <suppress files="OpenAiChatCompletionParameters.java" checks="TodoComment" lines="95" />
-  <suppress files="OpenAiChatMessage.java" checks="TodoComment" lines="255,271" />
```

Comment:
(Explanation)

* `OpenAiChatCompletionParameters` no longer has a todo (?)
* additional `@Beta` line moves the TODO down by two lines.